commcare_connect/templates/tailwind/components/cards/user_status_card.html

```diff
@@ -0,0 +1,29 @@
+<div class="bg-white shadow-sm p-4 rounded-2xl flex flex-col gap-4 w-110">
+  <div class="flex justify-between items-center text-gray-400 text-sm">
+    <h3 class="text-base font-medium text-brand-deep-purple">User Status Details</h3>
+    <div>
```

Comment:
we should implement the suspend button, and remove the status parts below

Reply:
Done. [c095c66](https://github.com/dimagi/commcare-connect/pull/571/commits/c095c668b3895abb6a4c3bf6dfc831c88a6465bd)